tests/python/gaia-ui-tests/gaiatest/tests/functional/persona/test_persona_app.py

```diff
@@ -46,7 +46,7 @@ def test_persona_standard_sign_in(self):
         assertionUtil = AssertionUtil()
         unpacked = assertionUtil.unpackAssertion(assertion)
 
-        self.assertEqual(AUDIENCE, unpacked['payload']['aud'])
+        self.assertTrue(AUDIENCE in unpacked['payload']['aud'])
```

Comment:
[assertIn](https://docs.python.org/2.7/library/unittest.html?highlight=assert#unittest.TestCase.assertIn) will give a better display if the test fails. Otherwise, we'll just get something like: `Expected false to be true`